pylmod/__init__.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+"""
+PyLmod is a module that implements MIT Learning Modules API in python
+"""
+VERSION = "0.0.1"  # pragma: no cover
+
```

Comment:
You are also going to want to import all the relevant classes here (only one so far, but probably more to come), i.e.

```python
from pylmod.client import Client

__all__ = ['Client']
```

so that importing is nicer on used projects, i.e.:
```python
import pylmod
gradebook = pylmod.Client(...)
```